src/style.css

```
@@ -0,0 +1,28 @@
+.starts {
+  display: flex;
+}
+
+.stars__star {
+  width: 50px;
+  height: 50px;
+  background-image: url(images/star.svg);
+  margin: 0;
+  flex: 1;
+}
+
+.body {
```

selector for body would look a lot better at the top of the document

Btw it is not affecting anything because you are targeting `body` class. Remove the dot and it will be applied to body tag ;)
```suggestion
body {
```